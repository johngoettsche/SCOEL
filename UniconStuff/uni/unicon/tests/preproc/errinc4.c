/* test error message: bad file name */

#include 77
