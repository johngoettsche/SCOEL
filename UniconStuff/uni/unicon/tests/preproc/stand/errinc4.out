pp: File errinc4.c; Line 3: invalid include file syntax
